cms/static/js/collections/group.js

```diff
@@ -5,17 +5,20 @@ function (_, str, Backbone, gettext, GroupModel) {
     'use strict';
     var GroupCollection = Backbone.Collection.extend({
         model: GroupModel,
-        comparator: 'order',
+
         /*
-         * Return next index for the model.
-         * @return {Number}
+         * Sort groups by name, alphabetically.
          */
-        nextOrder: function() {
-            if(!this.length) {
+        comparator: function(first, second) {
```

Comment:
Good catch, and thanks for the explanation. I tried to understand `sortBy` yesterday, but the documentation wasn't clear to me.